client/src/components/Export/ExportToZipButton/ExportToZipButton.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+import * as JSZip from 'jszip';
+import * as JSZipUtils from 'jszip-utils';
+import { saveAs } from 'file-saver';
+import { parse as toCSV } from 'json2csv';
+import { getS3ImageNameFromObjectUrl } from '../../../utils/api-util';
+
+/**
+ * Presentational component for a button that exports data to CSV
+ */
+export default class ExportToZipButton extends React.Component {
+
+    async handleClick() {
+        const { fields, data, exportTitle, images } = this.props;
+
+        const csv = toCSV(data, {fields});
+        const exportFileName = exportTitle.endsWith('.csv') ? exportTitle : exportTitle + '.csv';
+
+        const zip = new JSZip();
+        zip.file(exportFileName, csv);
+
+        for (const image of images) {
```

Comment:
If the no memorial in the query has an image, this errors. add a check to make sure images is not null.